package/models.py

```diff
@@ -158,6 +158,12 @@ def get_opengraph_image_url(self):
     def is_deprecated(self):
         return self.date_deprecated is not None
 
+    @property
+    def is_favorite(self):
```

Comment:
What do you think about calling this `has_favorites`?

Reply:
Done  in d505400e3aadf30fbb6537183f5ac2be0398ed1a @saadmk11